data/cli/jaeger-collector.yaml

```diff
@@ -0,0 +1,198 @@
+name: jaeger-collector
```

Comment:
we will need these to be versioned

Reply:
Yeah, for sure. This PR is largely here to demonstrate why I think the fully automated approach won’t work (notice the output of the command descriptions).

Reply:
The main yaml files are not rendered (those without extra command). If they look good we can tolerate odd output of the env command, since it is artificially pre-formatted.